src/main/getDiForUnitTesting.ts

```diff
@@ -124,6 +124,16 @@ export function getDiForUnitTesting(opts: { doGeneralOverrides?: boolean } = {})
   di.preventSideEffects();
 
   if (doGeneralOverrides) {
+    const globalOverrideFilePaths = getGlobalOverridePaths();
+
+    const globalOverrides = globalOverrideFilePaths.map(
+      (filePath) => require(filePath).default,
+    );
+
+    globalOverrides.forEach(globalOverride => {
+      di.override(globalOverride.injectable, globalOverride.overridingInstantiate);
+    });
+
```

Comment:
Note: This is here so that we can stop bloating this file. We can now just add file with postfix `.global-override-for-injectable.ts` which is used for override automatically.

Reply:
In future PR, we can consolidate all overrides here to new way.

Reply:
Is this worth it though since the end game would be to have as few of these global overrides as possible?